.circleci/config.yml

```diff
@@ -188,6 +188,7 @@ commands:
                       -e libtest \
                       -e obsolete \
                       -e future \
+                      -e TODO \
```

Comment:
@testautomation  excluding TODO tags from execution does not exclude these tests from the report. I have to post process Robot's output.xml to remove placeholders from report/log.